app.js

```diff
@@ -0,0 +1,15 @@
+const navSlide = () => {
+  const burger = document.querySelector('.humberger-menu');
+  const nav = document.querySelector('.primary-navigation');
+
+  burger.addEventListener('click', () => {
+    nav.classList.toggle('nav-active');
+    burger.classList.toggle('toggle');
+  });
+  document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click', () => {
+    nav.classList.remove('nav-active');
+    burger.classList.remove('toggle');
+  }));
+};
```

Comment:
The menu is working perfectly 🔥 Good job guys 👌

Reply:
Thank you so much